crates/circuit/src/nlayout.rs

```diff
@@ -105,6 +105,7 @@ impl PhysicalQubit {
         layout.phys_to_virt[self.index()]
     }
 }
+unsafe impl ::bytemuck::NoUninit for PhysicalQubit {}
```

Comment:
I don't think this is actually needed anymore (not sure when it was). Testing it locally nothing fails if you remove this.

```suggestion
```

Reply:
Removed!